R/ignore.R

```diff
@@ -22,5 +22,14 @@ use_build_ignore <- function(files, escape = TRUE, base_path = ".") {
 escape_path <- function(x) {
   x <- gsub("\\.", "\\\\.", x)
   x <- gsub("/$", "", x)
+
+  # at this point the string replacement for an absolute windows path would
+  # approximately be: C:\Users\hadley\...
+  # this causes the PCRE regex compilation error, which wants
+  # C:\\Users\hadley\...
```

Comment:
I still don't understand why you need an extra `\` here. Can you please rewrite the comment to specifically to address that? Ideally it would be at most 3 lines.

Reply:
Given the pattern `<driveletter>:\user\...`, you get a PCRE regex compilation error. This is evident in https://github.com/hadley/pkgdown/issues/307 which was impetus for this PR